mkroosevelt.js

```diff
@@ -94,9 +94,13 @@ function createSampleApp() {
     console.error('There was an error in copying the sample app: ' + err.message);
   }
 
-  if (fs.existsSync(path.normalize(cmd + '/.npmignore'))) {
+  try {
+    fs.accessSync(fs.existsSync(path.normalize(cmd + '/.npmignore')));
     fs.renameSync(path.normalize(cmd + '/.npmignore'), path.normalize(cmd + '/.gitignore')); // fix to compensate for this "feature" https://github.com/npm/npm/issues/1862
   }
+  catch (err) {
+    
```

Comment:
I left it blank because I was unsure. If it errors it really doesn't matter to the user it just meant they didn't have a .npmignore file. The sample app will still copy fine. Would what you like exactly?